cmd/litcli/ln.go

```diff
@@ -356,13 +356,12 @@ func sendPayment(ctx *cli.Context) error {
 			"%w", err)
 	}
 
-	// We use a constant amount of 500 to carry the asset HTLCs. In the
+	// We use a constant amount of 354 to carry the asset HTLCs. In the
```

Comment:
I don't think we should repeat constants in comments. Because if we change the constant then it's easy to forget to update the comment. Maybe:
```
// Use smallest HTLC sat amount to carry the asset HTLCs.
```

Reply:
Or maybe call out where the `DefaultOnChainHtlcAmount` value is set?